src/cryptography/hazmat/primitives/ciphers/algorithms.py

```diff
@@ -12,6 +12,9 @@
 
 
 def _verify_key_size(algorithm, key):
+    # Verify that the key is instance of bytes
+    utils._check_bytes("{0} key".format(algorithm.name), key)
```

Comment:
Let's drop the fancier error message and just use `key`, it'll be a small performance win, and it's consistent with all the other places we raise this error, and it should be obvious what key it is from context.

Reply:
Okay. Just "key" string.